Readme.MD

```diff
@@ -639,7 +639,7 @@ import { SearchBar } from 'react-native-elements'
 | placeholder | '' | string | set the placeholder text |
 | placeholderTextColor | '#86939e' | string | set the color of the placeholder text |
 | onChangeText | none | function | method to fire when text is changed |
-
+| clearIcon | { color: '#86939e', name: 'search' } | object {name (string), color (string), style (object)} | specify color, styling, or another [Material Icon Name](https://design.google.com/icons/) Note: pressing on this icon clears text inside the searchbar |
```

Comment:
Add punctuation between `..[Material Icon Name](https://design.google.com/icons/) Note: pressing..`

Maybe even put Note in a () for better readability.